SETUP.md

```diff
@@ -38,32 +38,34 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 1. Set up [nvm](https://github.com/creationix/nvm)
   1. Create nvm's working directory if it doesnt exist: `mkdir ~/.nvm`
   1. Add the following to `~/.bash_profile` or your desired shell configuration file:
-  
+
      ```
      # Load nvm function into the shell
      export NVM_DIR=~/.nvm
      source $(brew --prefix nvm)/nvm.sh
      ```
-     
+
   1. Pick up those changes: `source ~/.bash_profile`
 1. Install Node 6.9.0
-  1. We want to be on node 6.9.0 and npm 3.10.8.
+  1. We want to be on node 6.9.0 and npm 3.10.8 and yarn 0.16.1
   1. `nvm install 6.9.0 && nvm alias default 6.9` this command should make this version the default version and print something like: `Creating default alias: default -> 6.9.0 (-> v6.9.0)`
-  1. (You can reinstall with your updated version after you clone the repository if necessary) Reinstall node_modules `cd apps; rm -rf node_modules && npm install; cd ..`
+  1. `npm install -g yarn`
+  1. (You can reinstall with your updated version after you clone the repository if necessary) Reinstall node_modules `cd apps; yarn; cd ..`
 1. (El Capitan) Ensure that openssl is linked: `brew link --force openssl`
 1. Check that you have the correct versions of everything:
   1. Open a new Terminal window
   1. `ruby --version  # --> ruby 2.2.3`
   1. `nvm ls          # --> v6.9.0`
   1. `node --version  # --> v6.9.0`
   1. `npm --version   # --> 3.10.8`
+  1. `yarn -V         # --> 0.16.1`
 
 ### Ubuntu 14.04
 
 1. `sudo apt-get update`
 1. `sudo apt-get install -y git mysql-server mysql-client libmysqlclient-dev libxslt1-dev libssl-dev zlib1g-dev imagemagick libmagickcore-dev libmagickwand-dev openjdk-7-jre-headless libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev curl pdftk enscript libsqlite3-dev phantomjs build-essential`
   * **Hit enter and select default options for any configuration popups, leaving mysql passwords blank**
-1. Upgrade npm to 2.0. If `npm -v` says less than 2.0 then
+1. Upgrade npm to 3.10.8 If `npm -v` says less than 3.10.8 then
```

Comment:
It probably doesn't matter. I just noticed that the npm versions we were talking about in this README were inconsistent between ubuntu and mac instructions, so I updated it. Also it seems like the easiest way to install yarn is with npm (which I find rather amusing), so maybe having a later npm version is a good idea?